devito/ops/types.py

```diff
@@ -104,3 +105,13 @@ def as_coeff_Add(self):
         return sympy.S.Zero, self
 
     __repr__ = __str__
+
+
+class OpsStencil(basic.LocalObject):
```

Comment:
Will it be better to have them as OPS_stencil?

Reply:
No. No underscores in class names